dashboard/modules/reporter/reporter_agent.py

```diff
@@ -206,13 +209,24 @@ class ReporterAgent(
     def __init__(self, dashboard_agent):
         """Initialize the reporter object."""
         super().__init__(dashboard_agent)
-        if IN_KUBERNETES_POD:
```

Comment:
Do we have a test that actually runs with a docker container? If not, can we add one?

Reply:
I can look into that to validate resource limits in some cases -- we have a bit of local K8s infra and some fake docker provider tests which can be used to run tests.

I don't feel like doing full e2e testing of cgroups v1, v2 right now, though.

Reply:
Will add a couple lines to the KubeRay integration tests, since that's what I'm most familiar with.

Reply:
Added to KubeRay e2e test. The KinD environment in the CI probably uses cgroupsv1, not sure. Can think about testing both v1 and v2 later.